signalr/hubs/samples/7.x/SignalRHubsSample/wwwroot/js/chat.js

```diff
@@ -0,0 +1,41 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const connection = new signalR.HubConnectionBuilder()
+    .withUrl("/chat")
+    .build();
+
+  connection.on("ReceiveMessage", (user, message) => {
```

Comment:
If the intent is to have a runnable app, we'll need to add javascript code for the client results
https://learn.microsoft.com/aspnet/core/signalr/hubs?view=aspnetcore-7.0#typescript-client

Reply:
@BrennanConroy, sorry, I should have been more clear.  I just took your code that was inline from the doc.  I was hoping you could create runnable versions of them so we have one runnable sample somewhere.  I do not have enough experience with SignalR to make your original samples work.  Sorry for the confusion.